resources/views/checkout/steps/billing-address.blade.php

```diff
@@ -16,7 +16,7 @@
     mutate-event="setBillingAddressOnCart"
     v-slot="{ mutate, variables }"
 >
-    <div partial-submit="mutate">
+    <div partial-submit v-on:partial-submit="(ev) => mutate().then(ev.detail.resolve).catch(ev.detail.reject)">
```

Comment:
`e` vs `ev` vs `event`? Very next line we use `e` here, and other places we use `event`. It's already not ideal to be inconsistent here, but I'm especially not keen on adding yet another variation.

Reply:
I will update the functions to use `e`
`event` is a [reserved keyword](https://github.com/AnanthaRajuC/Reserved-Key-Words-list-of-various-programming-languages/blob/master/language-files/JavaScript%20Reserved%20Words.md#:~:text=escape-,event,-F) so while it works it's not good practice to use it.

Sidenote: It's fine in JS files if used correctly since it will compile variables down to shorter names.